Application/EdFi.Ods.Api/Security/AuthorizationStrategies/Relationships/Filters/RelationshipsAuthorizationFilters.cs

```diff
@@ -78,14 +72,15 @@ private static FilterApplicationDetails CreateClaimValuePropertyFilter(string pr
                     new FilterApplicationDetails(
                         "SchoolIdToStudentUSI",
                         @"StudentUSI IN (
-                        SELECT {newAlias1}.StudentUSI 
-                        FROM auth.SchoolIdToStudentUSI {newAlias1} 
-                        WHERE {newAlias1}.SchoolId IN (:SchoolId))",
+                            SELECT {newAlias1}.StudentUSI 
+                            FROM auth.StudentUSIToEducationOrganizationId {newAlias1} 
+                            WHERE {newAlias1}.SourceEducationOrganizationId IN (:SourceEducationOrganizationId))",
```

Comment:
Same here, you've modified the parameter name to force it to be SourceEducationOrganizationId.